test/run.sh

```diff
@@ -5,11 +5,12 @@ output=$(mktemp)
 trap 'rm -- "$output"' EXIT
 NODE_ENV=test karma start | tee "$output"
 # Search for: warnings from console.warn(), including Vue warnings; Sass
-# warnings; and warnings from Karma.
-if grep -F -e 'WARN LOG:' -e 'ERROR LOG:' -e 'Module Warning' -e 'WARN [web-server]:' -C5 "$output"; then
-	# Reset the text format in case the search results contained formatted text.
-	tput sgr0
-	echo
-	echo 'All tests passed, but there were warnings: see above.'
-	exit 1
+# warnings; and warnings from Karma. Ignore build warnings
```

Comment:
If we were to merge #1217 and address getodk/central#1029, would that resolve all the build warnings?

Reply:
Hmm maybe not, I think there'd still be this warning:

```
WARN LOG: 'Feature flag __VUE_PROD_HYDRATION_MISMATCH_DETAILS__ is not explicitly defined.
```

Reply:
I think it is possible for there to be build warnings that we'd want to be informed about. At the same time, we need tests to pass despite these warnings. What do you think about having test/run.sh count the number of warnings and assert that it's less than some fixed number? (3?)

Reply:
build warning tolerance was added in https://github.com/getodk/central-frontend/pull/1218